src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CachePoolPass.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\DefinitionDecorator;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class CachePoolPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method process() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.](https://www.codacy.com/app/git_9/symfony/file/2331802099/issues/source?bid=3207821&fileBranchId=3207834#l27)